openstef/data_classes/prediction_job.py

```diff
@@ -81,6 +82,10 @@ class PredictionJobDataClass(BaseModel):
     """Only required for create_wind_forecast task"""
     hub_height: Optional[float]
     """Only required for create_wind_forecast task"""
+    pipelines_to_run: PipelineType = PipelineType.BOTH
+    """The pipelines to run for this pj"""
+    forecast_pid: Optional[Union[int, str]]
```

Comment:
forecast_pid seems a bit too generic to me, to avoid confusion what do you think of renaming "forecast_pid" to "alternative_forecast_model_pid". I think this describes the id better.

Reply:
Yes you are right, I'll clarify the name.